### PR TITLE
azure - update azure package metadata

### PR DESCRIPTION
**Pull request by aluong**
Issue #2500 

- updated c7n_azure package long description (content based off `Azure Getting Started`)
- updated c7n_mailer package long description's content type

Sample package in test.pypi.org

- https://test.pypi.org/project/c7n-azure/
- https://test.pypi.org/project/c7n-mailer/

### CODE REVIEW

**Comment by stefangordon** [comment]
Just in time for 0.8.29 ;)

File: tools/c7n_azure/readme.md [hunk 1]
```diff
@@ -1,49 +1,64 @@
...
+Create a file named ``custodian.yml`` with this content:
+
+    policies:
+        - name: my-first-policy
+          description: |
+            Adds a tag to all virtual machines
+          resource: azure.vm
+          actions:
+            - type: tag
+              tag: Hello
```
Comment (kapilt) [inline tools/c7n_azure/readme.md]: looks good, and this is just docs, but fwiw, its also nice to support tagging multiple values in a single action impl

File: tools/c7n_azure/readme.md [hunk 2]
```diff
@@ -1,49 +1,64 @@
...
+First, choose one of the supported authentication mechanisms and either log in to Azure CLI or set
+environment variables as documented in [Authentication](http://capitalone.github.io/cloud-custodian/docs/azure/authentication.html#azure-authentication).
+
+    custodian run --output-dir=. custodian.yml
+
+
+If successful, you should see output similar to the following on the command line
+
+    2016-12-20 08:35:06,133: custodian.policy:INFO Running policy my-first-policy resource: azure.vm
+    2016-12-20 08:35:07,514: custodian.policy:INFO policy: my-first-policy resource:ec2 has count:1 time:1.38
```
Comment (kapilt) [inline tools/c7n_azure/readme.md]: on the azure example, should probably have a resource that corresponds to azure in the log output.

File: tools/c7n_azure/readme.md (hunk 1 above)
Comment (stefangordon) [reply tools/c7n_azure/readme.md]: Oh yeah that action also accepts `tags: { dictionary }`

File: tools/c7n_azure/readme.md (hunk 2 above)
Comment (stefangordon) [reply tools/c7n_azure/readme.md]: tx fixed

**Comment by stefangordon** [comment]
merging blocked on some sort of github issue even after attempt to restart builds with a new commit.... not sure...

**Pull request merged**